emr/application/controllers/Screeners.php

```diff
@@ -176,7 +176,8 @@ public function added_time() {
           'start' => $morn_time[$i],
           'end' => $end_date
         );
-        $this->db->insert('availability', $morn);
+          $this->load->model('Availability_model');
```

Comment:
You should do this in the autoload.php file on line 137 instead of loading the model each time

Reply:
Done!

Reply:
Looks good!